src/memory_region/mod.rs

```diff
@@ -20,8 +23,14 @@ pub trait MrAccess: Sync + Send + Debug {
     /// Get the remote key
     fn rkey(&self) -> u32;
 
-    /// Get the access of mr
-    fn access(&self) -> ibv_access_flags;
+    /// Get the `ibv_access_flags` of mr
+    fn ibv_access(&self) -> ibv_access_flags;
```

Comment:
Should we return a rdma_sys type?

Reply:
Exposing rdma_sys type is usually not a good idea, but we can use it to do some access checks quickly without the need of importing rdma_sys type.

High-level access API is also exposed, and it has a little overhead for converting. Users can choose one of them.

Reply:
Can we find a way to speed up the converting between these two types?

Reply:
One way is to enumerate all the possible combinations "automatically" via macro. The drawback of this method is the code size.